docs/part-3-serve-and-deploy-the-model/chapter-14-continuous-deployment-of-the-model-with-the-cicd-pipeline.md

```diff
@@ -812,37 +893,39 @@ latest version is consistently available on the Kubernetes server for use.
 
 === ":simple-github: GitHub"
 
-    In the **Actions** tab, click on the **Call Deploy** > **deploy**.
+    In the **Actions** tab, click on the **MLOps** workflow.
 
 === ":simple-gitlab: GitLab"
 
     You can see the pipeline running on the **CI/CD > Pipelines** page. Check the
-    `deploy` job:
+    `MLOps` pipeline.
 
-The output should look like this.
+The output should look like this:
 
+```text
+Run kubectl apply \
+deployment.apps/celestial-bodies-classifier-deployment configured
+service/celestial-bodies-classifier-service configured
 ```
-> mlem deployment run --load service_classifier --model model
 
-⏳️ Loading model from model.mlem
-⏳️ Loading deployment from service_classifier.mlem
-```
+If you execute the pipeline a second time, you should see the following output:
 
-Note that since the model has not changed, MLEM has not re-deployed the model.
+```text
+Run kubectl apply \
+deployment.apps/celestial-bodies-classifier-deployment configured
+service/celestial-bodies-classifier-service unchanged
+```
 
-??? bug "Having a `NameError: name 'UUID' is not defined` error? Read this!"
+As you can see, the deployment was successful and the service was unchanged.
```

Comment:
Unfortunately, I got a failure in the pipeline log:

```
from server for: "kubernetes/service.yaml": services "celestial-bodies-classifier-service" is forbidden: User "google-service-account@<my GCP project ID>.iam.gserviceaccount.com" cannot get resource "services" in API group "" in the namespace "default": requires one of ["container.services.get"] permission(s).
```
(replaced my actual GCP project ID for anonymity).
Did I miss something here?

Reply:
I did have to tinker with permissions, I might have missed to copy a specific one. I'll check.